### PR TITLE
Worker and ControlRpc status updates

### PR DESCRIPTION
**Pull request by alari**
Return worker status from `/status` endpoint, tendermint block from `/status/tendermint` endpoint;

Pass ControlStatus case class via ControlRpc.

### CODE REVIEW

File: node/src/main/scala/fluence/node/workers/WorkersHttp.scala [hunk 1]
```diff
@@ -155,9 +158,18 @@ object WorkersHttp {
           withWorker(appId)(w => workerApi.query(w, data, path, id).flatMap(tendermintResponseToHttp(appId, _)))
         }
 
-      case GET -> Root / LongVar(appId) / "status" ⇒
+      case GET -> Root / LongVar(appId) / "status" :? QueryWait(wait) ⇒
```
Comment (folex) [inline node/src/main/scala/fluence/node/workers/WorkersHttp.scala]: What is wait? Let's add some doc

File: node/src/main/scala/fluence/node/workers/WorkersHttp.scala [hunk 2]
```diff
@@ -155,9 +158,18 @@ object WorkersHttp {
...
         LogFactory[F].init("http" -> "status", "app" -> appId.toString) >>= { implicit log =>
-          withWorker(appId)(w => workerApi.status(w).flatMap(tendermintResponseToHttp(appId, _)))
+          withWorker(appId)(
+            _.services
+              .status(wait.filter(_ < 10).fold(1.second)(_.seconds))
```
Comment (folex) [inline node/src/main/scala/fluence/node/workers/WorkersHttp.scala]: Why 10? What's happening here? Maybe a comment would help

File: statemachine/control/src/main/scala/fluence/statemachine/control/ControlStatus.scala [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import io.circe.{Decoder, Encoder}
+import io.circe.generic.semiauto._
+
+/**
+ * Statemachine's status, fetched via Control RPC.
+ *
+ * @param expectEth Whether this statemachine expects to get Ethereum blocks or not
+ */
+case class ControlStatus(
+  expectEth: Boolean
```
Comment (folex) [inline statemachine/control/src/main/scala/fluence/statemachine/control/ControlStatus.scala]: `expectsEth` would be better, I think

File: node/src/main/scala/fluence/node/workers/WorkersHttp.scala (hunk 2 above)
Comment (alari) [reply node/src/main/scala/fluence/node/workers/WorkersHttp.scala]: Tbh I have no reason either for 10 nor for 1 second. Just two constants that "should be long enough". But I have added a comment.

**Pull request merged**